docs/deployment-raft.md

```diff
@@ -0,0 +1,115 @@
+# Orchestrator deployment: raft
+
+This text describes deployments for [orchestrator on raft](raft.md).
+
+This complements general [deployment](deployment.md) documentation.
+
+### Backend DB
+
+You may choose between using `MySQL` and `SQLite`. See [backend configuration](configuration-backend.md).
+
+- For MySQL:
+  - The backend servers will be standalone. No replication setup. Each `orchestrator` node will interact with its own dedicated backend server.
+  - You _must_ have a `1:1` mapping `orchestrator:MySQL`.
+  - Suggestion: run `orchestrator` and its dedicated MySQL server on same box.
+  - Make sure to `GRANT` privileges for the `orchestrator` user on each backend node.
+
+- For `SQLite`:
+  - `SQLite` is bundled with `orchestrator`.
+  - Make sure the `SQLite3DataFile` is writable to the `orchestrator` user.
```

Comment:
Suggestion: writable _by_
Question: does orchestrator try to ensure this file has any specific permissions?